examples/common/python/config/config.py

```diff
@@ -27,9 +27,14 @@
 import re
 import toml
 from string import Template
-from utility.file_utils import find_file_in_path
+from utility.file_utils import find_file_in_paths
 
-__all__ = [ "ConfigurationException", "parse_configuration_files", "parse_configuration_file" ]
+__all__ = [ "ConfigurationException", 
+    "parse_configuration_files",
+    "parse_configuration_file",
+    "read_config_from_toml" ]
+
+TCFHOME = os.environ.get("TCF_HOME")
```

Comment:
As you have removed the default relative path, now you should check if TCF_HOME is defined as environment variable or not. If not, then raise exception.
try:  
   os.environ["TCF_HOME"]
except KeyError: 
   print "Please set the environment variable TCF_HOME"
   sys.exit(1)

Reply:
Updated this to handle the error case. Rather than using exit, raising a KeyError.